canonical-web-and-design/pythonoperatorframework.io
# Update homepage copy

## Done

Updated the content on the homepage according to issue #13 

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://0.0.0.0:8047/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the content matches the request in the issue #13 

## Issue / Card
Fixes #13
